languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -1424,6 +1424,21 @@ USA
             <!-- Created by Tiago F. Santos, Portuguese rule, 2017-06-11 -->
             <url>https://pt.wikipedia.org/wiki/V%C3%ADcio_de_linguagem#Prolixidade_ou_preciosismo</url>
             <short>Expressão prolixa</short>
+
+            <rule>
```

Comment:
```suggestion
            <rule default="temp_off">
```

Reply:
Ahhhh… right! Sorry! I have committed it again!

Reply:
FYI when I add a suggestion, you don't have to commit it yourself, you can just click the 'Commit suggestion' button on GitHub ;)